yarn-project/end-to-end/src/composed/e2e_persistence.test.ts

```diff
@@ -61,14 +61,13 @@ describe('Aztec persistence', () => {
   beforeAll(async () => {
     dataDirectory = await mkdtemp(join(tmpdir(), 'aztec-node-'));
 
-    const initialContext = await setup(0, { dataDirectory, numberOfInitialFundedAccounts: 3 }, { dataDirectory });
+    const initialContext = await setup(1, { dataDirectory, numberOfInitialFundedAccounts: 3 }, { dataDirectory });
```

Comment:
This is not a necessary change and I did it before reverting some changes that made it necessary. But it's good to keep it like that because now setup waits for an empty block to be mined if there are no accounts deployed to ensure the network processes past genesis. Hence by using the setup to deploy the account we avoid waiting for the empty block to be mined.